### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by nelsonic**
You know, to make things (*A LOT*) simpler.
+ "switcher" intelligently routes requests to the correct scraper based on url  
(_which makes building a crawler *much* easier!_)
+ considerably fewer 404 tests requiring round-trips to GitHub! (#**NoBrainer**)

Reduced redundant tests from:
scraper-code-coverage

to: 
gs-code-coverage

(_focus on the number of **branches**_ ...)

***Way*** more maintanable!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,12 +154,300 @@ Sample output:
...
+   'izaakrogan', 'lynnaloo', 'nvcexploder', 'cwaring', 'missinglink', 'alanshaw', 'olizilla',
+   'tancredi', 'Ericat', 'pgte' 'hyprstack', 'iteles' ],
+  url: 'https://github.com/nelsonic/following?page=3',
+  next_page: 'https://github.com/nelsonic/following?page=4'
+}
+```
+
+### Starred Repositories
+
+The list of projects a person has *starred* a fascinating source of insight.
```
Comment (iteles) [inline README.md]: Missing word 'is' : "... has *starred* _is_ a fascinating..."

Comment (nelsonic) [inline README.md]: meh. happy to accept commits/PRs in the future if people want to get free points.

Comment (iteles) [inline README.md]: I'm happy to not point out things like this in future :+1:
I always read the readme updates before looking at the code so point out anything that breaks the flow of the readme for people reading it seeing as I'm going through it anyway. Usually an easy thing to just add to the next thing you work on.

Won't do it anymore though :+1:

**Comment by iteles** [comment]
I <3 this, it works everything out for itself. 
Can I merge now all checks are complete?

**Comment by nelsonic** [comment]
@iteles when ever *you* are ready...

**Pull request merged**